src/helpers/Monitor.cpp

```diff
@@ -235,6 +235,20 @@ void CMonitor::onConnect(bool noRule) {
         }
     }
 
+    Debug::log(LOG, "checking if we have seen this monitor before: {}", szName);
+    // if we saw this monitor before, set it to the workspace it was on
+    if (g_pCompositor->m_mseenmonitorworkspacemap.contains(szName)) {
```

Comment:
eeee did you even compile and test this? This won't compile as you forgot the capitalization

Reply:
I'll click the re-request button when ready

no I didn't - I made the change and pushed so I could test on my personal machine when I get a minute - I can't build Hyprland on macos.

Reply:
also you can highlight code with the suggest change button in the conversation and suggest a change

![image](https://github.com/user-attachments/assets/4b0171a5-7a82-456a-bb21-95bf3dea0b04)

might save some back and forth